web/src/lib/components/photos-page/asset-grid.svelte

```diff
@@ -447,25 +546,33 @@
   {/if}
 
   {#if element}
-    <slot />
-
-    <!-- (optional) empty placeholder -->
+    <div style:visibility={showSkeleton ? 'hidden' : 'visible'}>
```

Comment:
consider putting this on the asset-grid - or even wrap the grid/scrollbar - assuming intersection-obs still works